bravado/http_future.py

```diff
@@ -112,23 +115,21 @@ def __init__(self, future, response_adapter, operation=None,
             also_return_response_default=False,
         )
 
-    def response(self, timeout=None, fallback_result=None, exceptions_to_catch=FALLBACK_EXCEPTIONS):
+    def response(self, timeout=None, fallback_result=SENTINEL, exceptions_to_catch=FALLBACK_EXCEPTIONS):
         """Blocking call to wait for the HTTP response.
 
         :param timeout: Number of seconds to wait for a response. Defaults to
             None which means wait indefinitely.
         :type timeout: float
-        :param fallback_result: callable that accepts an exception as argument and returns the
-            swagger result to use in case of errors
-        :type fallback_result: callable that takes an exception and returns a fallback swagger result
+        :param fallback_result: either the swagger result or a callable that accepts an exception as argument
+            and returns the swagger result to use in case of errors
+        :type fallback_result: either any type of object that represents a valid Swagger result or a callable that
```

Comment:
This seems to be the same as in param, could we define the type in a more accurate way?
maybe with typing annotation style?